plugins/utils/neorg.nix

```diff
@@ -120,23 +125,26 @@ in
             level = cfg.logger.level;
 
             modes =
-              attrsets.mapAttrsToList
-              (mode: modeConfig: {
-                name = mode;
-                inherit (modeConfig) hl;
-                level = let
-                  level = modeConfig.level;
-                in
-                  if isNull level
-                  then null
-                  else
-                    (
-                      if (isInt level)
-                      then level
-                      else helpers.mkRaw "vim.log.levels.${strings.toUpper level}"
-                    );
-              })
-              cfg.logger.modes;
+              if (isNull cfg.logger.modes)
+              then null
+              else
+                attrsets.mapAttrsToList
+                (mode: modeConfig: {
+                  name = mode;
+                  inherit (modeConfig) hl;
+                  level = let
+                    level = modeConfig.level;
+                  in
+                    if isNull level
+                    then null
+                    else
+                      (
+                        if (isInt level)
+                        then level
+                        else helpers.mkRaw "vim.log.levels.${strings.toUpper level}"
+                      );
```

Comment:
This should not be needed as the level can't be null

Reply:
Indeed. Fixed !